README.md

```diff
@@ -37,10 +37,10 @@ gulp.task('translate', ['build'], function(){
 module.exports = gulp;
 ```
 
-### What happens on disk
+### What happens:
 
-The source is copied for each language present in the locale directory. Every
-file is pruned for translation calls.
+`po` files are catalogued/merged from each locale directory, then used
```

Comment:
Merging of po files is standard behavior when defining multiple catalogs.

The purpose of this section is to document how the plugin is unique, specifically that it creates an individual build for each target language, translates on disk, and removes the translation infrastructure from the source.  (maybe it should be said kinda like so)